vllm/model_executor/layers/attention.py

```diff
@@ -141,6 +141,8 @@ def forward(
                 else:
                     input_metadata.attn_bias = _make_alibi_bias(
                         self.alibi_slopes, batch_size, seq_len, query.dtype)
+                if self.num_kv_heads != self.num_heads:
+                    input_metadata.attn_bias._bias = input_metadata.attn_bias._bias.unflatten(1, (self.num_kv_heads, self.num_queries_per_kv))
```

Comment:
@megha95 Could you elaborate more on this case? I think the current RoPE-based GQA models do not need this. Is this for GQA + ALiBi?